### ISSUE TITLE
update and uninstall configs

### ISSUE THREAD

**Issue by dalekunce** [AmericanRedCross/posm#46]
what if settings change. we need an update and uninstall config

**Comment by jflemer-ndp** [comment]
Agree.  For the moment, most items will appropriately reconfigure themselves if you simply update `settings.local` and re-run `bootstrap.sh ...`.  There are certainly a few areas that don't do that cleanly (e.g. `demo_data` for macrocosm is one I know of).  If you run across any specific ones that trip you up, post them on this ticket.

Moving to chef, ansible, deb packages, or some combination thereof would probably be the path forward.

**Comment by mojodna** [comment]
Short-term, we may be able to put re-configuration in an upstart task set to `start on started hostname` or similar.
